index.js

```diff
@@ -35,62 +35,75 @@ var REPO_PROPOSALS_BASE_URL = GITHUB_BASE_URL + 'apple/swift-evolution/blob/mast
  *
  * `className`: Mapping of states in the proposals JSON to the CSS class names used
  * to manipulate and display proposals based on their status.
+ * 
+ * `count`: Number of proposals that determine after all proposals is loaded
  */
 var states = {
   '.awaitingReview': {
     name: 'Awaiting Review',
     shortName: 'Awaiting Review',
-    className: 'awaiting-review'
+    className: 'awaiting-review',
+    count:0
```

Comment:
Please add a space after each colon for all `count:0` lines (x11).

```suggestion
    count: 0
```

Reply:
@benrimmington All right.